projects/commudle-admin/src/app/components/home/components/home-events/home-events-card/home-events-card.component.html

```diff
@@ -0,0 +1,27 @@
+<div *ngIf="community && event" class="home-events-card">
+  <div *ngIf="registrations > 0" class="home-events-attending">
+    <nb-icon icon="people"></nb-icon>
+    {{registrations}} Registered
+  </div>
+  <div [routerLink]="['/communities', community.slug, 'events', event.slug]" class="home-events-image clickable">
+    <img *ngIf="event.header_image_path" alt="{{event.name}}" src="{{event.header_image_path}}">
+    <div *ngIf="!event.header_image_path">
+      <app-skeleton-text [rows]="1"></app-skeleton-text>
+    </div>
+  </div>
+  <div class="home-events-date">
+    {{event.start_time | date:'MMM d'}}
+  </div>
+  <div class="home-events-details">
+    <p [routerLink]="['/communities', community.slug, 'events', event.slug]" class="home-events-name clickable">
+      {{event.name | truncatetext:35}}
+    </p>
+    <div class="home-events-host">
+      <nb-user [name]="community.name" [picture]="community.logo_path" [routerLink]="['/communities', community.slug]"
+               [showTitle]="false"></nb-user>
```

Comment:
Let's use our own image tag, nb-user fails to position images properly sometimes

Reply:
Done!